app/views/assessments/show.html.erb

```diff
@@ -138,8 +137,8 @@
 
   <%# Display any attachments %>
   <% if @assessment.attachments.length > 0 %>
```

Comment:
Then students would see the 'Add ... Attachment' link if there were more than 0 attachments.